src/index.html

```diff
@@ -31,31 +33,32 @@ <h1 class="h-center">Why am I seeing this?</h1>
               </p>
             </div>
             <div class="dcs-full-page__action">
-              <a href="https://globalclimatestrike.net" class="dcs-button">
-                Join the strike →
+              <a href="https://globalclimatestrike.net" class="dcs-full-page__button">
+                Join the strike <span class="dcs-button__icon">➔</span>
               </a>
             </div>
           </div>
         </div>
       </div>
     </main>
-    <footer class="dcs-footer sc-dcs">
+    <footer class="dcs-footer">
```

Comment:
I removed common styles to isolate components -> easier to know what you're changing